src/test/deploy/functions/ensureCloudBuildEnabled.ts

```diff
@@ -71,63 +71,57 @@ describe("checkRuntimeDependencies()", () => {
     timeStub.withArgs("motd.cloudBuildErrorAfter").returns(errorAfter);
   }
 
-  ["nodejs10", "nodejs12", "nodejs14"].forEach((runtime) => {
-    describe(`with ${runtime}`, () => {
-      describe("with cloudbuild service enabled", () => {
-        beforeEach(() => {
-          mockServiceCheck(true);
-        });
+  describe("with cloudbuild service enabled", () => {
+    beforeEach(() => {
+      mockServiceCheck(true);
+    });
 
-        it("should succeed", async () => {
-          stubTimes(Date.now() - 10000, Date.now() - 5000);
+    it("should succeed", async () => {
+      stubTimes(Date.now() - 10000, Date.now() - 5000);
 
-          await expect(checkRuntimeDependencies("test-project", runtime)).to.eventually.be
-            .fulfilled;
-          expect(logStub?.callCount).to.eq(0);
-        });
-      });
+      await expect(ensureCloudBuildEnabled("test-project")).to.eventually.be.fulfilled;
+      expect(logStub?.callCount).to.eq(0);
+    });
+  });
 
-      describe("with cloudbuild service disabled, but enabling succeeds", () => {
-        beforeEach(() => {
-          mockServiceCheck(false);
-          mockServiceEnableSuccess();
-          mockServiceCheck(true);
-        });
+  describe("with cloudbuild service disabled, but enabling succeeds", () => {
+    beforeEach(() => {
+      mockServiceCheck(false);
+      mockServiceEnableSuccess();
+      mockServiceCheck(true);
+    });
 
-        it("should succeed", async () => {
-          stubTimes(Date.now() - 10000, Date.now() - 5000);
+    it("should succeed", async () => {
+      stubTimes(Date.now() - 10000, Date.now() - 5000);
 
-          await expect(checkRuntimeDependencies("test-project", runtime)).to.eventually.be
-            .fulfilled;
-          expect(logStub?.callCount).to.eq(1); // enabling an api logs a warning
-        });
-      });
+      await expect(ensureCloudBuildEnabled("test-project")).to.eventually.be.fulfilled;
+      expect(logStub?.callCount).to.eq(1); // enabling an api logs a warning
+    });
+  });
 
-      describe("with cloudbuild service disabled, but enabling fails with billing error", () => {
-        beforeEach(() => {
-          mockServiceCheck(false);
-          mockServiceEnableBillingError();
-        });
+  describe("with cloudbuild service disabled, but enabling fails with billing error", () => {
+    beforeEach(() => {
+      mockServiceCheck(false);
+      mockServiceEnableBillingError();
+    });
 
-        it("should error", async () => {
-          stubTimes(Date.now() - 10000, Date.now() - 5000);
+    it("should error", async () => {
+      stubTimes(Date.now() - 10000, Date.now() - 5000);
 
-          await expect(checkRuntimeDependencies("test-project", runtime)).to.eventually.be.rejected;
-        });
-      });
+      await expect(ensureCloudBuildEnabled("test-project")).to.eventually.be.rejected;
```

Comment:
Nit since you weren't really changing this code: This probably should be calling rejectedWith() and asserting that we throw an error that mentions billing. Same applies for the permissions error test